presentations/proposed/scipy2023/proposal.md

```diff
@@ -0,0 +1,86 @@
+# SciPy 2022 Proposal
+
+Jul 11-16, 2023
+
+https://www.scipy2023.scipy.org/present
+
+## Title
+
+Scientific Python---Year in Review
+
+## Session type
+
+Talk (30 minutes)
+
+## Abstract (100 words)
+
+The Scientific Python project aims to better coordinate the ecosystem and grow the community.
+This talk focuses on our efforts to better coordinate project development, and to improve shared infrastructure.
+Come hear about our recent progress and our plans for the coming year!
+
+## Description (500 words)
+
+The Scientific Python project's vision is to help pave the way toward a vibrant, unified scientific Python community.
+It focuses its efforts along two primary axes: _(i)_ to create a joint community around all scientific Python projects
+and _(ii)_ to support maintainers by building cross-cutting technical infrastructure and tools.
+
+Last year we launched the project with new websites, a Hugo web theme, a social media campaign, and a collaborative coordination process similar to PEPs called SPECs.
+This year, we are fortunate to have received [funding from CZI](https://scientific-python.org/grants/community_and_communications_infrastructure/) for the continued development, maintenance, and support of web and documentation themes, as well as other community infrastructure, in collaboration with Quansight.
+With the community and communication infrastructure having support for the next few years, we are able to focus more on technical topics and the SPECs.
+
+As a first project, we are [funded to work on improving sparse *array*](https://scientific-python.org/grants/sparse_arrays) (vs matrix) semantics in SciPy with the goal of removing sparse *matrices* and, eventually, also NumPy *matrices* from several ecosystem libraries. In line with our philosophy of continually working with the community and incorporating their feedback, we hosted the first of several [sparse summits](https://scientific-python.org/summits/sparse/)—virtual meetings to identify sparse array needs in ecosystem libraries.
+This project spans multiple projects, including numpy, scipy, scikit-image, networkx, scikit-learn, and many of the packages built on top of these libraries.
+
+In addition to the sparse summit, we have hosted a [domain stack summit](https://scientific-python.org/summits/domain-stacks/), to discuss domain-specific umbrella projects that host several others, as well as the first [annual developer summit](https://scientific-python.org/summits/developer/).
+This in-person workshop brought together over 30 community members for a week-long, collaborative sprint, and tackled topics including build & testing systems, continuous integration infrastructure, release management tools, and community management.
+
+Finally, we will update the community on our progress on the [decadal plan](https://scientific-python.org/grants/planning_next_decade/).
+
```

Comment:
All our efforts until now have helped to bring the Scientific Python community together and have resulted in joint efforts for tool development and shared infrastructure that will impact positively the whole ecosystem. There is still a long road ahead but we are ready to prepare scientific Python for the next decade.